unison-cli/src/Unison/CommandLine.hs

```diff
@@ -125,95 +106,6 @@ emojiNote lead s = P.group (fromString lead) <> "\n" <> P.wrap s
 nothingTodo :: (ListLike s Char, IsString s) => P.Pretty s -> P.Pretty s
 nothingTodo = emojiNote "😶"
 
-completion :: String -> Line.Completion
```

Comment:
Most of these were just moved to the `Completion` module. I'll note which ones were removed (because they're unused)